src/src/cron/prepareCertificateHtml.cron.ts

```diff
@@ -75,6 +76,7 @@ export class PrepareCertificateHtmlCron {
 						context,
 						context_id,
 					);
+				console.log('usrAttendanceList list', usrAttendanceList);
```

Comment:
remove console logs pls

Reply:
console log on dev is required for testing and track cron job flow. after testing complete on dev server then i will remove all logs and sent PR again